apps/web/src/pages/ApplicantDashboardPage/ApplicantDashboardPage.tsx

```diff
@@ -100,16 +93,13 @@ export const DashboardPage = ({
               data-h2-flex-direction="base(column)"
               data-h2-gap="base(x1)"
             >
-              <ReviewApplicationPreviewList
-                applicationsQuery={unpackMaybes(currentUser?.poolCandidates)}
-              />
-              {/* Temporary separator till https://github.com/GCTC-NTGC/gc-digital-talent/issues/10772 */}
-              <Separator data-h2-margin="base(0)" decorative />
-              <ReviewRecruitmentProcessPreviewList
-                recruitmentProcessesQuery={unpackMaybes(
-                  currentUser?.poolCandidates,
-                )}
-              />
+              {currentUser?.poolCandidates ? (
+                <ApplicationsProcessesTaskCard
+                  applicationsProcessesTaskCardQuery={unpackMaybes(
+                    currentUser?.poolCandidates,
+                  )}
+                />
+              ) : null}
```

Comment:
Is there a null state for this?

Reply:
I can't find a  design for a null state and I don't think one exists on purpose because the components within the `ApplicationsProcessesTaskCard` component handle the empty/null states. Are you suggesting removing the ternary altogether?

Reply:
830bd400f1e5957dd7cf78a26aabf30b1065dc43